show-land/sass/layout/_hero.scss

```diff
@@ -0,0 +1,58 @@
+.text-box {
+  position: absolute;
+  top: 30%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  text-align: center;
+}
+
+.header-primary {
+  &__sub {
+    display: block;
+    text-transform: uppercase;
+    font-size: $fontSizeBase;
+  }
+
+  &__main {
+    display: block;
+    font-size: 5rem;
```

Comment:
You can create a new variable for this in the shared variables file: 
`$fontSize8Xl: 5em;`

Reply:
ok